### ISSUE TITLE
chip: chip group should have toolbar role

### ISSUE THREAD

**Issue by hellogreg** [patternfly/patternfly-elements#2666]
## Description of the issue

Chip groups should have a `toolbar` role to communicate to assistive tech users that they have entered a group of controls

### Impacted component(s)

- `pf-chip`
- `pf-chip-group`

### Steps to reproduce

1. Go to [chip documentation](https://patternflyelements.org/components/chip/).
2. Enable screen reader (I initially used Mac VoiceOver with Safari).
3. Navigate to a chip group using the `tab` key.

### Expected behavior

The user receives some indication that they have entered a group of controls and should now use arrow keys to navigate within the chip group.
